planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -358,6 +366,8 @@ void planning_scene_monitor::PlanningSceneMonitor::scenePublishingThread()
             if (octomap_monitor_) lock = octomap_monitor_->getOcTreePtr()->reading();
             scene_->getPlanningSceneMsg(msg);
           }
+          // also publish timestamp of this robot_state
+          msg.robot_state.joint_state.header.stamp = last_robot_motion_time_;
```

Comment:
nice catch!
I consider it quite a shortcoming that MoveIt! does rarely consider time stamps..